### PR TITLE
Update How-to Guide landing page

### LINKED ISSUE #106: [how-to guides] Refactor landing page
Replace contribution guidelines with overview of guides and use component to link to categories.

### PR DESCRIPTION
**Pull request by sacha-l**
This creates a top-level category view for How-to guides. I've removed the content that was there previously because that is covered in #122. 

Closes #106.

### CODE REVIEW

File: v3/how-to-guides/index.mdx [hunk 1]
```diff
@@ -1,120 +1,62 @@
...
+  link={`/how-to-guides/storage-migrations`}
+  
+/>
+<RelatedMaterialBlock
+  text={`Discover different ways to implement consensus mechanisms in your runtimes.`}
+  linkText={`Consensus`}
+  link={`/how-to-guides/consensus`}
+/>
+<RelatedMaterialBlock
+  text={`All things releated to integrating and extending parachain capabilities to standalone Substrate chains.`}
```
Comment (jimmychu0807) [inline v3/how-to-guides/index.mdx:52]:
```suggestion
  text={`All things related to integrating and extending standalone Substrate chains to parachains.`}
```

File: v3/how-to-guides/index.mdx [hunk 2]
```diff
@@ -1,120 +1,62 @@
...
+
+<RelatedMaterialBlock
+  text={`Learn the simple patterns that runtime engineers know inside out. These guides cover the basics 
+  that will help you understand more complex topics.`}
+  linkText={`Basics`}
+  link={`/how-to-guides/basics`}
+  />
+<RelatedMaterialBlock
+  text={`A collection of guides that cover common concepts for building pallets using FRAME, with specific use cases to 
+  give meaningful context to each guide.`}
```
Comment (jimmychu0807) [inline v3/how-to-guides/index.mdx:23]:
```suggestion
  text={`A collection of best practices on building pallets using FRAME.`}
```

File: v3/how-to-guides/index.mdx [hunk 3]
```diff
@@ -1,120 +1,62 @@
...
+/>
+<RelatedMaterialBlock
+  text={`All things releated to integrating and extending parachain capabilities to standalone Substrate chains.`}
+  linkText={`Parachains`}
+  link={`/how-to-guides/parachains`}
+  
+/>
+<RelatedMaterialBlock
+  text={`Guides relating to using tools for getting your chain ready for production or integrate ways to interact with
+  your runtime.`}
```
Comment (jimmychu0807) [inline v3/how-to-guides/index.mdx:59]:
```suggestion
  text={`Guides for tools that are not included out-of-the-box to help you managing Substrate chains in production.`}
```

**Review by jimmychu0807 — APPROVED** [review]

File: v3/how-to-guides/index.mdx [hunk 4]
```diff
@@ -1,120 +1,60 @@
...
-      description:
-        'Code-based examples that make use of this guide. This shows at least one reference of what this guide covers with a working example.This could be a reference to some existing codebase within Substrate or not; or new code that lives in the how-to guide repo.',
-    },
-    {
-      title: 'Resources',
-      description:
-        'A bulleted list of links to similar guides; other devhub ressources; and related material. For example, other how-to guides; tutorials; knowledgebase articles; or Rust docs.',
-    },
-  ]}
+Substrate how-to guides are in-depth guides developers with some prior knowledge of Substrate and a
```
Comment (lsgunnlsgunn) [inline v3/how-to-guides/index.mdx:10]: ... guides for developers ...

File: v3/how-to-guides/index.mdx [hunk 5]
```diff
@@ -1,120 +1,60 @@
...
+  text={`All things related to integrating and extending standalone Substrate chains to parachains.`}
+  linkText={`Parachains`}
+  link={`/how-to-guides/parachains`}
+  
+/>
+<RelatedMaterialBlock
+  text={`Guides for tools that are not included out-of-the-box to help you managing Substrate chains in production.`}
+  linkText={`Tools`}
+  link={`/how-to-guides/tools`}
+/>
```
Comment (lsgunnlsgunn) [inline v3/how-to-guides/index.mdx:60]: We can iterate on the wording, but ideally we'd have parallel construction for each category (probably starting with a verb).

**Review by lsgunnlsgunn — COMMENTED** [review]
Sorry I'm a little late to the party. Just a couple of notes.

**Pull request merged**